composer.json

```diff
@@ -94,9 +94,15 @@
       "BigBlueButton\\": ["src", "tests/integration"]
     }
   },
+  "autoload-dev": {
+    "psr-4": {
+      "BigBlueButton\\Tests\\Functional\\": ["tests/functional"]
```

Comment:
Note: My plan is to do such Namespace restructuring also for `unit` and `integration` tests later. The reason is to improve DX on IDE integration (e.g. for PHPStorm, which automatically derives the namespace from the composer.json)